### DISCUSSION TITLE
Bring your own service account

### DISCUSSION THREAD

**Discussion by AmitArie** [strimzi/strimzi-kafka-operator#8780] [Q&A] [answered]
Hi,
I have an existing service account in my EKS cluster and I want to use it as a service account for my `KafkaConnect` cluster.
is there a way to do it?
The documentation says that the `serviceAccount` property is not a string but `ResourceTemplate`.

The rational is to use a service account that is being mapped to AWS IAM Role in order to have the ability to use S3 Sink ( Confluent ).
Thanks!

**Comment by scholzj** [comment]
No, you cannot use your own Service Account.You can only configure some additional labels and annotations for the Service Account managed by Strimzi.

**Reply by AmitArie** [reply]
Thanks for the reply!
Any idea why is that?
What's the rational?

**Reply by scholzj** [reply, marked as answer]
I would probably ask it the other way around. 😉 You do not bring in your services, pods, secrets, config maps etc. So why service accounts? Also, the operator needs to understand how the operand is deployed and configured, it needs to make sure it has the required RBAC rights etc.

**Reply by AmitArie** [reply]
@scholzj makes sense, thanks.
